File: dataset/java_mix/C06WordcountType1B.java
import java.util.HashMap;
import java.util.Map;
public class C06WordcountType1B {
    // Added comments and white spaces
    public static Map<String, Integer> wordCount(String s) {
        Map<String, Integer> counts = new HashMap<>();
        
        for (String w : s.split("\\s+")) {
            counts.put(w, counts.getOrDefault(w, 0)+1) ;
        }

        return counts;
    }
}

File: dataset/javascript/C06WordcountType1B.js
// Added comments and white spaces
function wordCount(s) {
    const counts = {};
    
    for (const w of s.trim().split(/\s+/)) {
        counts[w] = (counts[w] ?? 0) + 1;
    }

    return counts;
}

module.exports = { wordCount };
